packages/conversation/src/models/v1/contact-info-message/contact-info-message.ts

```diff
@@ -0,0 +1,103 @@
+/**
+ * Message containing contact information.
+ */
+export interface ContactInfoMessage {
+
+  /** @see ContactInfoMessageContactInfoMessage */
+  contact_info_message?: ContactInfoMessageItem;
+}
+
+export interface ContactInfoMessageItem {
+  /** @see NameInfo */
+  name: NameInfo;
+  /** Phone numbers of the contact */
+  phone_numbers: PhoneNumberInfo[];
+  /** Physical addresses of the contact */
+  addresses?: AddressInfo[];
+  /** Email addresses of the contact */
+  email_addresses?: EmailInfo[];
+  /** @see OrganizationInfo */
+  organization?: OrganizationInfo;
+  /** URLs/websites associated with the contact */
+  urls?: UrlInfo[];
+  /** Date of birth in YYYY-MM-DD format. */
+  birthday?: string;
+}
+
+/**
+ * Name information of the contact.
+ */
+export interface NameInfo {
+  /** Full name of the contact. Will be the displayed contact name. Format as desired. */
+  full_name: string;
+  /** First name/given name. */
+  first_name?: string;
+  /** Family name/last name/surname. */
+  last_name?: string;
+  /** Middle name. */
+  middle_name?: string;
+  /** Prefix before the name. e.g. Mr, Mrs, Dr etc. */
+  prefix?: string;
+  /** Suffix after the name. e.g. Jr or PhD */
+  suffix?: string;
+}
+
+/**
+ * Phone numbers of the contact.
+ */
+export interface PhoneNumberInfo {
+  /** Phone number with country code included. */
+  phone_number: string;
+  /** Phone number type, e.g. WORK or HOME. */
+  type?: string;
```

Comment:
No, I think it corresponds to the field you have on a UI where you are asked to give a nickname to a phone number when you have a contact with multiple phone numbers: It could be work, home, mobile, school, fax... or any custom label.